ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -64,7 +77,7 @@ sub handle_config_file
         backup => '.old'
     );
 
-    foreach my $option_set (qw(comment_options options)) {
+    foreach my $option_set (qw(comment_options options multiline_options)) {
```

Comment:
My recollection is that add_or_replace_lines looked great for single line options, but I couldn't see a way for it to easily work for multiline options. Maybe I'm missing something though.

You provide add_or_replace_lines with a regex of lines to match, and what to replace it with. So if you provide the option name as the match, this is going to match every instance of that option, and replace each line with the same value right? That's why I went for the remove and readd approach - I know the multiline options will always end up at the end of the file, but as we're doing it in sorted key order, the output should be consistent at least so I didn't see a problem.